crates/storage/src/lazy/lazy_cell.rs

```diff
@@ -161,7 +161,10 @@ where
             false => {
                 // Clear without loading from storage:
                 let footprint = <T as SpreadLayout>::FOOTPRINT;
-                assert!(footprint <= 16); // magic number
+                assert!(
+                    footprint <= crate::traits::FOOTPRINT_CLEANUP_THRESHOLD,
+                    "The footprint limit is violated for this type!"
```

Comment:
```suggestion
                    "cannot clean-up a storage entity with a footprint of {}. maximum threshold for clean-up is {}.",
                    footprint,
                    footprint_threshold,
```

Reply:
also make `footprint_threshold` a binding to `crate::traits::FOOTPRINT_CLEANUP_THRESHOLD`